### ISSUE TITLE
Executes the GTFS Validator after downloading a dataset

### ISSUE THREAD

**Issue by davidgamez** [MobilityData/mobility-feed-api#228]
### Describe the problem

As part of the Datasets metadata, we would like to include a summary of the information produced by the [GTFS validator](https://github.com/MobilityData/gtfs-validator). This issue implements a process to execute the validator after downloading a dataset. 

### Scope
- Executes GTFS validator after a dataset is downloaded
- Copy validator's output files(report.json, reports.hml and system_errors.json) to a specified GCP bucket with a parameterized filename format

### Out of Scope
- Update MobilityData API Database with the validation reports

This issue depends on #225 

### Proposed solution

There are two main solutions that we can explore:

1. Create a cloud function that contains the validator's _executable_ 
2. Create a cloud function that makes requests to the [GTFS Web Validator](https://gtfs-validator.mobilitydata.org/)

**Comment by cka-y** [comment]
Tasks:
- [ ] Investigate possible implementation
- [ ] Share with the team and come up with a decision
- [ ] POC
- [ ] Create new issue for the implementation

**Comment by emmambd** [comment]
Completed - new issue created with #339
